src/styles/_colors.scss

```diff
@@ -0,0 +1,6 @@
+$black: #161722;
```

Comment:
It's better to rename files without `_` prefix. In rare occasions we would prefix it with `_`.